### DISCUSSION TITLE
Quartz.NET 3.6

### DISCUSSION THREAD

**Discussion by lahma** [quartznet/quartznet#1905] [Announcements]
This release contains new API to reset errored trigger state in job store, some bug fixes and refinement of package dependencies/targets.

__NEW FEATURES__

* Add explicit netcoreapp3.1 and net6.0 targets to MS integration projects (#1879)
* Use IHostApplicationLifetime instead of IApplicationLifetime in >= netcoreapp3.1 Hosting targets (#1593)
* Add ResetTriggerFromErrorState functionality (#1904)

__FIXES__

* Fix named connection string resolution when using MS DI and its configuration system (#1839)
* Upgrade to System.Configuration.ConfigurationManager 6.0.1 to avoid vulnerable dependency chain (#1792)
* Fix configuration handling for custom DB provider (#1795)
* Add extra overloads for registering listeners (#1852)
* JobDataMap.TryGetGuidValue should return Guid instead of int (#1856)
* Upgrade to Newtonsoft.Json 13.0.1 (#1859)

**Full Changelog**: https://github.com/quartznet/quartznet/compare/v3.5.0...v3.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/quartznet/quartznet/releases/tag/v3.6.0'>Quartz.NET 3.6</a>.</em>

**Comment by s3YwCf2ZbfJG4SHAfjQMAjtsf** [comment]
Can you pin this please? Thanks :) (Have it at the top of the list always until the next version)
